acceptance/tests/vault/vault_test.go

```diff
@@ -163,9 +186,6 @@ func TestVault(t *testing.T) {
 	require.NoError(t, err)
 
 	consulHelmValues := map[string]string{
-		// TODO: Update the global image once 1.11 is GA.
```

Comment:
Removed as this is no longer needed and also prevents us from running -enable-enterprise tests with the proper image.